### PR TITLE
Minor wording/formatting changes

### PR DESCRIPTION
**Pull request by orangesamus**

### CODE REVIEW

File: tokenomics-explainers/004-rework-support.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ _Figure 1 - Profitability with and without MEV theft_
...
...ontract. This churn, along with the potential for NO supply to outpace rETH demand (due to things like the introduction of ETH-Only validators) could lead to a significant NO queue. To support decentralization, protocol safety, and the health of the Ethereum network the Saturn Upgrade also introduces an express queue. To enter the express queue you must use an express queue ticket, and tickets are distributed to favor both small NOs and existing NOs. To favor small NOs, two[^1] express queue tickets will be given to all NOs (including new NOs). Express queue tickets will also be given to all existing NOs (_enough tickets to allow for express migration of all bonded ETH to 4-ETH bond validators[^1]). The express queue works as a parallel queue that moves twice* as fast as the standard NO queue. ETH from the deposit pool is matched with validator deposits by taking alternating turns matching from the express queue and the standard queue. On each express queue turn two[^1] validator matches will be made, but on each standard queue turn only one validator match is made (this is what causes the express queue to move twice[^1] as fast). If a queue is empty, those matches will be skipped.
```
Comment (orangesamus) [inline tokenomics-explainers/004-rework-support.md]: Fixed twice* to twice[^1]

File: tokenomics-explainers/001-why-rework.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ Unfortunately, over the last few years, we've seen that these ideals do not play
 
 ### Running minipools is primarily Speculative
 
-Unfortunately, the staking requirement in **Ideal 1** makes the financial success of running minipools overwhelmingly dominated by RPL performance.
```
Comment (orangesamus) [inline tokenomics-explainers/001-why-rework.md]: Wording was awkward to finish previous paragraph with "Unfortunately" and then start the next with "Unfortunately"

File: tokenomics-explainers/001-why-rework.md [hunk 3]
```diff
@@ -82,7 +82,7 @@ The 10% borrowed-ETH minimum is something the protocol tries to impose upon the
 
 Consider a mature case with a 3% solo APY and 1.5% RPL value loss per year due to supply inflation. Using the same assumptions as earlier, this results in a **2.3% drop** compared to solo staking rewards. In such a case, there is no reason to make minipools if you can afford solo validators.
 
-This does present a solution as well – the minimum staked RPL can be reduced. Note that this would reduce the RPL to rETH TVL relationship per **Ideal 3**, which means that the cost to avoiding this brittleness is letting go of a relationship that can be known ahead of time.
```
Comment (orangesamus) [inline tokenomics-explainers/001-why-rework.md]: Makes it seem like we are suggesting a fix by lowering the requirement. Instead we are just highlighting a theoretical fix we aren't actually going to implement due to the effects it has on "Ideal 3"

**Comment by LongForWisdom** [comment]
Ended up making a separate PR as I'm not using my main branch for this. Changes went in here: https://github.com/LongForWisdom/RPIPs/pull/70

**Pull request closed without merging**